AUTHORS

```diff
@@ -5,11 +5,25 @@ Michael Durrant <mdurrant-b3@users.noreply.github.com>
 Michael Durrant <mdurrant@a10networks.com>
```

Comment:
You can remove this file

Reply:
This is add by Hunter, maybe we can check this with him later.

Reply:
Yeah let's remove it. These are auto added or something due to pbr tool.

Reply:
removed